contracts/libs/Ibc.sol

```diff
@@ -286,7 +286,11 @@ library Ibc {
         outStr = string(buffer);
     }
 
-    function receiptRootKey(string memory clientId, uint256 height) internal pure returns (bytes memory proofKey) {
-        proofKey = abi.encodePacked("client/", clientId, "/receiptRoot/", toStr(height));
+    function receiptRootKey(bytes32 chainId, bytes32 clientType, uint256 height)
+        internal
+        pure
+        returns (bytes memory proofKey)
+    {
+        proofKey = abi.encodePacked("chain/", chainId, "/client/", clientType, "/receiptRoot/", toStr(height));
```

Comment:
need to verify this so we're in alignment across peptide/relayer/contracts.

see Jack's PR: https://github.com/polymerdao/polymerase/pull/2036/files?diff=split&w=1#diff-bf10a17910f29c052bfd060654113f39c7e9e6f1db4adcbc0538309413cd9501R77

Reply:
Let's use `chain/<chainId>/client/<clientType>/receiptRoot/<height>`

Reply:
also, the `chainID` and the `clientType` should be strings (as `height` is)

Reply:
I was assuming chainId was already going to be encoded as bytes, but we can leave that in for a future optimizaiton

Reply:
will change it to be back into a string